public/js/contribute/pad/render.js

```diff
@@ -1712,7 +1733,7 @@ function addMosaic(kwargs) {
   // THERE IS NO IMAGE YET
   if (
     objecttype === 'templated' &&
-    (page.activity === 'edit' || (page.activity === 'preview' && !src))
+    (page.activity === 'edit' || (page.activity === 'preview' && !src)) // TO DO: what is src?
```

Comment:
what is src? src doesn't exist elsewhere in the file (img.src is not accessible here either)